benches/note_decryption.rs

```diff
@@ -15,7 +15,11 @@ use zcash_note_encryption_zsa::{batch, try_compact_note_decryption, try_note_dec
 #[cfg(unix)]
 use pprof::criterion::{Output, PProfProfiler};
 
-fn bench_note_decryption<FL: OrchardFlavor>(c: &mut Criterion) {
+mod utils;
```

Comment:
When running `cargo bench`, I see tests for proving and verifying for Orchard and OrchardZSA.
I don't see the decryption tests. Are you sure it is activated?